src/pickers/DatePicker.js

```diff
@@ -69,6 +85,32 @@ class AdDatePicker extends Component {
       </FormControl>
     );
   }
+  else{
+    let nepaliDate = (!!this.state.value ? new NepaliDate(new Date(this.state.value)).format('YYYY-MM-DD'): new NepaliDate().format('YYYY-MM-DD'))
+    return (
+      <FormControl fullWidth={fullWidth}>
+        <label>{!!label ? formatMessage(intl, module, label) : null}</label>
+        <Calendar       
+          onChange={this.onChangeNepal}
+          defaultDate={nepaliDate} 
+          language="en"
+          style={{width:"100%", display: "flex", position:"static"}}
+          dateFormat="DD/MM/YYYY"
+          placeholder="Select date"
+        />
+      </FormControl>
+    );
+  }
+  }
 }
 
-export default injectIntl(withTheme(withStyles(styles)(AdDatePicker)));
+const mapStateToProps = (state) => ({
+    isSecondaryCalendarEnabled: state.core.isSecondaryCalendarEnabled ?? false,
+  });
+
+export default injectIntl(
+    withModulesManager(
+      withHistory(connect(mapStateToProps, {})(withTheme(withStyles(styles)(DatePicker)))),
```

Comment:
Consider using _null_ instead of empty object. It will clearly communicate that the component does not dispatch any actions to the Redux store.